Modules/Microsoft365DSC/DSCResources/MSFT_EXOQuarantinePolicy/MSFT_EXOQuarantinePolicy.schema.mof

```diff
@@ -0,0 +1,19 @@
+
+[ClassVersion("1.0.0.0"), FriendlyName("EXOQuarantinePolicy")]
+class MSFT_EXOQuarantinePolicy : OMI_BaseResource
+{
+    [Key, Description("The Identity parameter specifies the QuarantinePolicy you want to modify.")] String Identity;
+    [Write, Description("The EndUserQuarantinePermissionsValue parameter specifies the end-user permissions for the quarantine policy.")] SInt32 EndUserQuarantinePermissionsValue;
```

Comment:
Looks like a typo. Did you mean to define this as Int32 (not SInt32)?

Reply:
fixed